types/node/assert.d.ts

```diff
@@ -755,10 +763,31 @@ declare module "assert" {
          * );
          * // Throws: AssertionError: Got unwanted exception: Whoops
          * ```
+         *
+         * If an `TypeError` is thrown and his error is `TypeError` and message is 'Wrong Value'
```

Comment:
this cannot be found in original docs? Mind adding this instead to the tests? (and add one)